texttospeech/google/cloud/texttospeech_v1/types.py

```diff
@@ -21,13 +21,13 @@
 
 from google.cloud.texttospeech_v1.proto import cloud_tts_pb2
 
-_shared_modules = []  # pragma: NO COVER
```

Comment:
I added this because the coverage dropped after `shared_modules` were taken away. Do we typically drop the coverage threshold for unit tests instead? @tseaver

Reply:
Logically we should be excluding any generated code from coverage.  That particular line shouldn't need the pragma, but the loop below would (since there aren't any shared modules).  Could we update the code generator to omit these lines in that case?

Reply:
See: https://github.com/googleapis/gapic-generator/pull/2775